### ISSUE TITLE
Is this a permission issue?

### ISSUE THREAD

**Issue by justmy2centx** [dennisstritzke/ipsec_exporter#23]
I am getting this error when calling /metrics.

```INFO[0000] Listening on 9536                             source="serve.go:42"
WARN[0093] Unable to retrieve the status of tunnel 'fw1'. Reason: exit status 1  source="status.go:60"
WARN[0093] Unable to retrieve the status of tunnel 'fw2'. Reason: exit status 1  source="status.go:60"

**Comment by gunhu** [comment]
@justmy2centx Hi,

Yes it is :)

I will open an issue to get it fix (with a PR if possible)

**Comment by gunhu** [comment]
Here is the relevant associated PR:

https://github.com/dennisstritzke/ipsec_exporter/pull/27

**Comment by dennisstritzke** [comment]
Merged this change through 6cd3829.

**Comment by dennisstritzke** [comment]
The change is included in the [v0.4.0 release](https://github.com/dennisstritzke/ipsec_exporter/releases/tag/v0.4.0).
